src/components/header/header.tsx

```diff
@@ -18,55 +18,37 @@ const getTitle = () => {
   return title;
 };
 
-class Header extends React.Component<object, { title?: string }> {
-  constructor(public props: object) {
-    super(props);
-    this.state = {
-      title: getTitle(),
-    };
-  }
+const Header = () => {
+  const [titleValue, setTitle] = useState(getTitle());
```

Comment:
please, add type to useState when it is not obvious what type you will use there